server/.golangci.yml

```diff
@@ -86,9 +86,6 @@ issues:
         channels/api4/plugin.go|\
         channels/api4/plugin_test.go|\
         channels/api4/post_test.go|\
-        channels/api4/preference_test.go|\
-        channels/api4/reaction_test.go|\
```

Comment:
@TheInvincibleRalph That is the change in question. I think you introduced it via [`31a7b57` (#28618)](https://github.com/mattermost/mattermost/pull/28618/commits/31a7b57836fe883478688721c03429de68dd40d6).

Reply:
Oh, alright. I'll make the correction now. Thank you!